sdk/personalizer/ai-personalizer-rest/tests.yml

```diff
@@ -0,0 +1,22 @@
+# Parameterize Location to the test resources deployment script. This allows
+# specifying a different Azure region to use when manually triggering the live
+# tests pipeline. This is useful for when we want to test in different
+# environments: Prod, Canary, etc.
+parameters:
+- name: Location
+  displayName: Location
+  type: string
+  default: eastus2euap
+
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
```

Comment:
Do you need to also check in a bicep template? Otherwise what are the live tests testing?

Reply:
We have the `test-resources.json` checked-in already, that's not enough?

Reply:
Oh, somehow I didn't see that when I went looking for it 🤦